client/login/magic-login/request-login-email-form.jsx

```diff
@@ -161,6 +209,16 @@ class RequestLoginEmailForm extends Component {
 
 		return (
 			<div className="magic-login__form">
+				{ siteIcon && (
+					<div className="magic-login__form-header-icon">
+						<img
+							src={ siteIcon }
+							width={ 64 }
+							height={ 64 }
+							alt={ `${ this.state.site?.name } icon` }
```

Comment:
The site name is in the text so probably fine to just let this be hidden for screen-readers?

Reply:
Yep, we could just add aria-hidden=true. I'd be fine either way on that. I don't know off the top what best accessibility practice is for this case.